pos_product_template/__openerp__.py

```diff
@@ -75,5 +75,5 @@
     'images': [
         'static/src/img/screenshots/pos_product_template.png',
     ],
-    'installable': False,
+    'installable': True,
```

Comment:
@dansanti   Thanks for your contribution to the OCA. 
There is a little more than this key to change in openerp.py.
Let me introduce to you some basic information about contribution in OCA and I will review other points in more comments inside the code..
In general, please follow the principles: 
* Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
* Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
* We recommend to include a screenshot as well to help new comers (nice to have)
* Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3
* Please sort all imports in alphabetical order following: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports